spec/controllers/solidus_bolt/webhooks_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'SolidusBolt::WebhooksController', type: :request do
+  describe '#update' do
+    subject(:endpoint_call) { post "/webhooks/bolt", params: {} }
```

Comment:
You have not provided the headers[X-Bolt-Hmac-Sha256'] no where, so you are not really testing that the value of that specific header is pass to the sorter